packages/recomponents/src/components/r-tabs/r-tab.vue

```diff
@@ -1,16 +1,25 @@
 <template>
-    <div v-show='isActive'>
+    <div :id="computedPanelId"
+         role="tabpanel"
+         :aria-labelledby="computedTabId"
+         v-if="$slots.default"
+         v-show="isActive"
+    >
         <slot></slot>
     </div>
 </template>
 
 <script>
+    /* eslint-disable no-underscore-dangle */
```

Comment:
this `eslint` statement isn't required

Reply:
It is needed because of line 41 - I will change it to `eslint-disable-line` for more clarity